src/components/Recorder.js

```diff
@@ -92,6 +92,8 @@ export default class Recorder extends H5P.EventDispatcher {
 
     const loadAudioUrl = new Promise((resolve, reject) => {
       this.once('wav-delivered', e => {
+        console.log('Uploading audio to the server...')
```

Comment:
Maybe remove this console log if it's not needed?

Reply:
I think this console.log might be useful to help locate this customized code in the future. Otherwise, the next dev might have to go through all the files to find out this piece of code.